components/Timeline/TimelineEvent/TimelineEvent.js

```diff
@@ -8,8 +8,8 @@ TimelineEvent.propTypes = {
 
 function TimelineEvent({ content, title }) {
   return (
-    <div className={styles.eventContainer}>
-      <h4 className={styles.eventTitle}>{title}</h4>
+    <div className="ml-8 mr-0 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
```

Comment:
```suggestion
    <div className="mb-8 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
```

in a 4-digit value, it reads like a clock (top, right, bottom, left)

Reply:
done